src/lib.rs

```diff
@@ -1,4 +1,5 @@
 #![warn(missing_docs)]
+#![warn(clippy::debug_assert_with_mut_call)]
```

Comment:
Maybe link to that issue here? I guess `git blame` and looking up the commit/PR description would get there...